src/helm/benchmark/annotation/omnimath_annotator.py

```diff
@@ -0,0 +1,69 @@
+from typing import Any
+
+from helm.benchmark.adaptation.request_state import RequestState
+from helm.benchmark.annotation.annotator import Annotator
+from helm.clients.auto_client import AutoClient
+from helm.common.request import Request
+
+
+# Following https://github.com/KbsdJames/Omni-MATH/blob/main/GPT_eval/get_result.py
+def parse_report(report):
+    parts = report.split("## ")
+    data = {}
+    for part in parts[1:]:
+        lines = part.strip().split("\n")
+        title = lines[0].strip()
+        content = "\n".join(lines[1:]).strip()
+        if title == "Justification":
+            data[title] = content
+        else:
+            data[title] = lines[1].strip() if len(lines) > 1 else ''
+    return data
+
+
+class OmniMATHAnnotator(Annotator):
+    """The OmniMATH autograder."""
+
+    name = "omnimath"
+
+    def __init__(self, auto_client: AutoClient):
+        self._auto_client = auto_client
+        with open("src/helm/benchmark/annotation/omnimath/gpt_evaluation_template.txt") as f:
```

Comment:
There are two additional steps you need to take to make this work when someone uses `pip install` to install HELM:

1. Add a pattern to the [manifest](https://github.com/stanford-crfm/helm/blob/main/MANIFEST.in) that matches the path to `gpt_evaluation_template.txt`.
2. Use `importlib_resources` to open the file. Refer to [this example](https://github.com/stanford-crfm/helm/blob/ee10b8fd5a0f46949c98cd7a0a79cb7e1b163073/src/helm/benchmark/metrics/efficiency_metrics.py#L50-L54).

Reply:
addressed in the latest change